.github/workflows/mutate.yml

```diff
@@ -10,10 +10,10 @@ jobs:
     strategy:
       matrix:
         args:
-          - '--build-arg n_mutations=500 --build-arg modules=src/{burrow,cfmm,checker,checkerMain,common,constants}.ml'
-          - '--build-arg n_mutations=500 --build-arg modules=src/{fa2Interface,lqt,parameters}.ml'
-          - '--build-arg n_mutations=50  --build-arg modules=src/{liquidationAuction,sliceList,mem,ptr}.ml'
-          - '--build-arg n_mutations=50  --build-arg modules=src/avl.ml'
+          - '--build-arg n_mutations=500 --build-arg modules="src/burrow.ml src/cfmm.ml src/checker.ml src/checkerMain.ml src/common.ml src/constants.ml"'
+          - '--build-arg n_mutations=500 --build-arg modules="src/fa2Interface.ml src/lqt.ml src/parameters.ml"'
+          - '--build-arg n_mutations=50  --build-arg modules="src/liquidationAuction.ml src/sliceList.ml src/mem.ml src/ptr.ml"'
+          - '--build-arg n_mutations=50  --build-arg modules="src/avl.ml"'
```

Comment:
By the way I noticed yesterday that if you put the arguments _after the target_ you don't need the `--build-arg` bit. For example, locally I've been able to do the following:
```console
earthly +test-mutations --n_mutations=1000 --modules=src/parameters.ml
```
Just pointing this out in case it helps avoiding the duplication above.

Reply:
Oh wow, I didn't know that syntax existed. Thanks!